database/src/test/kotlin/no/nav/su/se/bakover/database/TestDataHelper.kt

```diff
@@ -1029,15 +981,17 @@ internal class TestDataHelper(
         vilkårsvurderinger: Vilkårsvurderinger,
         grunnlagsdata: Grunnlagsdata,
     ) {
-        bosituasjongrunnlagPostgresRepo.lagreBosituasjongrunnlag(behandlingId, grunnlagsdata.bosituasjon)
-        fradragsgrunnlagPostgresRepo.lagreFradragsgrunnlag(behandlingId, grunnlagsdata.fradragsgrunnlag)
-        when (vilkårsvurderinger) {
-            is Vilkårsvurderinger.Revurdering -> {
-                uføreVilkårsvurderingRepo.lagre(behandlingId, vilkårsvurderinger.uføre)
-                formueVilkårsvurderingPostgresRepo.lagre(behandlingId, vilkårsvurderinger.formue)
-            }
-            is Vilkårsvurderinger.Søknadsbehandling -> {
-                formueVilkårsvurderingPostgresRepo.lagre(behandlingId, vilkårsvurderinger.formue)
+        dataSource.withTransaction { tx ->
```

Comment:
Kunne kanskje slått sammen de to input-objektene til typen som aksepteres av det nye repoet og lagret via det?